mysql/README.md

```diff
@@ -0,0 +1,19 @@
+# MySQL Data Accelerator
```

Comment:
This looks like a Data Connector quickstart, not a Data Accelerator one

Reply:
Fixed